### ISSUE TITLE
Bulk import data

### ISSUE THREAD

**Issue by nshah14285** [valeriansaliou/sonic#168]
Hello,
I have a CSV file whose size is 528MB and I would like to insert that data in the Sonic to test the performance.
Is there a way to import whole data at once without using the node app?

Thanks.

**Comment by valeriansaliou** [comment]
Hello! For now there’s not, unless you code your import utility in your favorite alternative language in something else than JS for Node.

--

Valerian Saliou
CTO at Crisp

**Comment by nshah14285** [comment]
Is there a direct way to import into RocksDB with the correct sonic require format?

**Comment by valeriansaliou** [comment]
No, that's not going to work. As Sonic also needs to generate proper FSTs for the data you're importing.

**Comment by nshah14285** [comment]
OK Thanks.
